samples/photo_album/package.json

```diff
@@ -9,14 +9,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "blueimp-file-upload": "^10.2.0",
```

Comment:
@rebeccapeltz , why are those changes needed?

Reply:
security audit  - when you npm install you get security errors - this can be automated for a repo in github but I just updated it manually - I'm attaching a picture showing it automated

<img width="1171" alt="auto-security" src="https://user-images.githubusercontent.com/354444/71834250-9f41b300-3063-11ea-8af2-8f11c2b8ad81.png">

Reply:
If you're not using package-lock.json you can ignore the above.  I found that people were uncomfortable with code with npm security alerts in open source.